rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -259,6 +259,16 @@ impl UtxoCheckStatus {
     }
 }
 
+/// Relevant data for a checked UTXO. The uuid and kyt_provider are kept
+/// backward compatibility reasons. They should be set to `None` since
```

Comment:
```suggestion
/// Relevant data for a checked UTXO. The UUID and `kyt_provider` are kept for
/// backward-compatibility reasons. They should be set to `None` since
```